lib/rubocop/cop/style/space_around_keyword.rb

```diff
@@ -183,6 +183,7 @@ def space_after_missing?(range)
                           char == '('.freeze
           return false if accept_left_square_bracket?(range) &&
                           char == '['.freeze
+          return false if range.source_buffer.source[pos, 2] == '&.'.freeze
```

Comment:
I'd extract this check in a method. It's a bit obscure what we're checking for otherwise.